the-collab-lab/tcl-22-smart-shopping-list
# MUI theme and AddItems component styles

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description



This sets up a Material UI theme alongside the existing global styles and uses both to style the overall layout and the AddItems component. 

*Responsive styles coming soon*

## Related Issue


#14 

## Acceptance Criteria


* Material UI components are used
* /additems page looks like the desktop design in [Figma](https://www.figma.com/file/VH8Z8iouh7F9zBBv1kbuq5/Shopping-App-design?node-id=0%3A1)

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |
|  ✓  | :link: Update dependencies |

## Updates

### Before


![screencapture-tcl-22-shopping-list-web-app-additems-2021-05-28-01_25_51](https://user-images.githubusercontent.com/68360696/119934373-ab2d5e00-bf53-11eb-80cc-30a4a4766a19.png)

### After


![screencapture-localhost-3000-additems-2021-05-28-01_23_50](https://user-images.githubusercontent.com/68360696/119934310-9224ad00-bf53-11eb-9988-ceb8b79db246.png)


## Testing Steps / QA Criteria


To test the feature, use one of the following ways:

1. Follow the [preview link](https://tcl-22-shopping-list--pr59-dt-add-items-styles-670l0tmv.web.app)
2. Launch the app locally from your terminal:
- Pull down this branch with `git pull origin dt-add-items-styles` and check that branch out with `git checkout origin dt-add-items-styles`
- Run `npm install` to update dependencies, then `npm start` to launch the app in your browser at http://localhost:3000

In the app, navigate to Add Groceries view to see the changes.
